public/modules/custom/grants_oma_asiointi/src/Plugin/Block/AsiointirooliBlock.php

```diff
@@ -69,8 +61,8 @@ public static function create(
     array $configuration,
     $plugin_id,
     $plugin_definition,
-  ) {
-    return new static(
+  ): static {
```

Comment:
The return type hit (`static`) does not match with the actual return type (`self`)